[2025-02-12T23:00:29.256+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T23:00:29.404+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: workflow_1.print_date scheduled__2025-02-11T23:00:00+00:00 [queued]>
[2025-02-12T23:00:29.494+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: workflow_1.print_date scheduled__2025-02-11T23:00:00+00:00 [queued]>
[2025-02-12T23:00:29.523+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-12T23:00:29.734+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): print_date> on 2025-02-11 23:00:00+00:00
[2025-02-12T23:00:29.754+0000] {standard_task_runner.py:72} INFO - Started process 479 to run task
[2025-02-12T23:00:29.772+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'workflow_1', 'print_date', 'scheduled__2025-02-11T23:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/retry.py', '--cfg-path', '/tmp/tmp5upij0tc']
[2025-02-12T23:00:29.780+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask print_date
[2025-02-12T23:00:30.195+0000] {task_command.py:467} INFO - Running <TaskInstance: workflow_1.print_date scheduled__2025-02-11T23:00:00+00:00 [running]> on host aec89d06531b
[2025-02-12T23:00:30.505+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='workflow_1' AIRFLOW_CTX_TASK_ID='print_date' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-11T23:00:00+00:00'
[2025-02-12T23:00:30.507+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T23:00:30.554+0000] {logging_mixin.py:190} INFO - Today is 2025-02-12
[2025-02-12T23:00:30.555+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T23:00:30.586+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T23:00:30.588+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=workflow_1, task_id=print_date, run_id=scheduled__2025-02-11T23:00:00+00:00, execution_date=20250211T230000, start_date=20250212T230029, end_date=20250212T230030
[2025-02-12T23:00:30.759+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T23:00:30.823+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-12T23:00:30.838+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
